updated design onboarding (#91)
ubclaunchpad/docs: handbook/onboarding/design.md

### GitHub Teams

> Designers should have a presence in Github to know what is going on between product teams. First step would be to make a [Github](https://github.com/) account if you don't have one!

Reviewer: ```suggestion
Designers should have a presence in GitHub to know what is going on between product teams. First step would be to [make a GitHub account](https://github.com/join) if you don't have one!
```

> Second, Make sure you are a part of the GitHub team [`@ubclaunchpad/design`](https://github.com/orgs/ubclaunchpad/teams/design), and check that you have access to the [Design repository](https://github.com/ubclaunchpad/design) - this repository contains (some) design assets, resources, and tracking issues.

Reviewer: ```suggestion
Second, make sure you are a part of the GitHub team [`@ubclaunchpad/design`](https://github.com/orgs/ubclaunchpad/teams/design), and check that you have access to the [Design repository](https://github.com/ubclaunchpad/design) - this repository contains (some) design assets, resources, and tracking issues.
```

> Third, make sure that you are added into your team's Github repository by your team's tech lead. 

Reviewer: Should this be part of the checklist? I've left it out of the "Onboarding for Everyone" page because presumably this will be the leads' responsibility, and this onboarding will largely happen during/after team formation

Author: hmm yes I def see what you mean! yeah i guess we can remove it then!

> Last, familiarize yourself with Github and how it works. We will be using Github for most of our product management and asset sharing. (This is where we should consolidate all assets & asset links) You don't need to become a pro right away, most of your Github use will be with issues and pull requests. Check how those work here: [Github guide](../tools/github.md)

Reviewer: ```suggestion
Last, familiarize yourself with GitHub and how it works. We will be using GitHub for most of our product management and asset sharing (this is where we should consolidate all assets and asset links) You don't need to become a pro right away, most of your GitHub use will be with issues and pull requests - check how those work in our [GitHub guide](../tools/github.md).
```

Make sure all designers are given "editor" access to the [shared Launch Pad Design subfolder](https://drive.google.com/drive/u/0/folders/1Zfe25r3D77hGdyMkj0tlxHNa-r7fAq1d), which is owned by the `team@ubclaunchpad.com` account.

> Also, if your team decides to use google drive, make sure you are added to that by your project lead. Also make sure that you are linking your gdrive assets to your Github repository for easy accessibility.

Reviewer: ```suggestion
If you decide to use Google Drive with your team, make sure you add links to your Google Drive assets to your team's GitHub repository for easy accessibility.
```

### Figma

> Make sure that you have a Figma account which you can create here: [Figma](https://www.figma.com/). It's completely free to use, web based and its the best collaborative design tool out there!

Reviewer: ```suggestion
Make sure that you [have a Figma account](https://www.figma.com/). It's completely free to use, web based, and is Launch Pad's recommended collaborative design tool.
```

> * You can also download the Figma app on your desktop here: [Download Link](https://www.figma.com/downloads/)

Reviewer: ```suggestion

You can also download the Figma app on your desktop here: [Download Link](https://www.figma.com/downloads/)
```

Other alternatives available out there are Adobe XD and Sketch. If you plan to use any of those tools, it is recommended that you import your designs into Figma so that you can share your prototypes and wireframe files with your developers.

## Workflows

> The Agile Method is the workflow that teams in LaunchPad usually follow. It is important that you familiarize youself with this type of workflow especially working in cross-functional product teams. 

Reviewer: ```suggestion
The Agile Method is the workflow that teams in Launch Pad usually follow. It is important that you familiarize youself with this type of workflow especially working in cross-functional product teams - you can learn more about it in our [handbooks' sprint planning guide](../project-management/sprints.md).
```

😄

Author: Oh you got me there, forgot to add in the link!! Nice!! haha

> During the whole cycle, you will have to make sure that all your assets are constantly being uploaded into Github. (or gdrive links can be found in your project's Github repository)

Reviewer: ```suggestion
During the whole cycle, you will have to make sure that all your assets are either constantly being uploaded into GitHub OR that links to Google Drive or Figma can be found in your project's GitHub repository.
```
